openverse_catalog/dags/common/loader/loader.py

```diff
@@ -34,19 +34,22 @@ def load_s3_data(
     )
 
 
```

Comment:
Note for reviewers: in this file I deleted the `load_from_s3` method in favor of calling directly through to the `sql.load_s3_data_to_intermediate_table`, since it no longer has any additional logic.

Then I added the new `upsert_data` method.